integration_tests/src/main/python/dpp_test.py

```diff
@@ -364,7 +364,6 @@ def setup_tables(spark):
                                  " from {}) tmp".format(dim_table) +
                                  " ON f.hr = tmp.hr AND f.dt = tmp.dt WHERE tmp.ts < CURRENT_TIMESTAMP"),
         conf=dict(_dpp_conf + [('spark.sql.adaptive.enabled', aqe_enabled),
-                               ("spark.rapids.sql.castStringToTimestamp.enabled", "true"),
```

Comment:
Changes like this seems not necessary, personally it would be better to keep them unchanged to reduce the size of this PR.

Reply:
OK, will revert.